src/modules/services/n8n.nix

```
@@ -0,0 +1,86 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+
+with lib;
+
+let
+  inherit (lib) types;
+  cfg = config.services.n8n;
+  format = pkgs.formats.json { };
+  configFile = format.generate "n8n.json" cfg.settings;
+in
+{
+  options = {
+    services.n8n = {
+      enable = mkEnableOption "n8n";
+
+      address = lib.mkOption {
+        type = types.str;
+        description = "Listen address";
+        default = "127.0.0.1";
+        example = "127.0.0.1";
+      };
+
+      port = lib.mkOption {
+        type = types.port;
+        default = 5432;
```

The default in their docs is `5678`. `5432` is the default for postgres, so we should avoid using that.

```suggestion
        default = 5678;
```